app/src/main/java/com/jasonette/seed/Service/websocket/JasonWebsocketService.java

```diff
@@ -19,34 +17,34 @@
 /*****************************************
 
  ### Events:
-     - There are 4 events: $websocket.onopen, $websocket.onclose, $websocket.onmessage, $websocket.onerror
+ - There are 4 events: $websocket.onopen, $websocket.onclose, $websocket.onmessage, $websocket.onerror
```

Comment:
Ouch, all whitespace lost.  Makes this file harder to understand.

Reply:
This is ok